databuilder/query_engines/sqlite.py

```diff
@@ -80,7 +82,15 @@ def get_sql(self, node):
 
     @get_sql.register(Value)
     def get_sql_value(self, node):
-        return sqlalchemy.literal(node.value)
+        return sqlalchemy.literal(self.convert_value(node.value))
+
+    def convert_value(self, value):
+        if isinstance(value, BaseCode):
+            return value.value
```

Comment:
Another alternative to mentioning `BaseCode` here would be to maintain a registry of mappings that was populated from outside.

Reply:
I don't feel strongly which of the three options you take, but we should keep an eye on this and consider refactoring later.

Reply:
OK, that sounds sensible. I just wanted to make sure there wasn't some Obviously Correct approach I was missing.